packages/@lwc/integration-karma/scripts/karma-configs/test/tags.js

```diff
@@ -11,12 +11,21 @@ const {
     COMPAT,
     SYNTHETIC_SHADOW_ENABLED,
     FORCE_NATIVE_SHADOW_MODE_FOR_TEST,
+    ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE,
+    ENABLE_SCOPED_CUSTOM_ELEMENT_REGISTRY,
+    DISABLE_ARIA_REFLECTION_POLYFILL,
+    NODE_ENV_FOR_TEST,
 } = require('../../shared/options');
 
+// These are used to decide the directory that coverage is written to
 const TAGS = [
     `${SYNTHETIC_SHADOW_ENABLED ? 'synthetic' : 'native'}-shadow`,
     FORCE_NATIVE_SHADOW_MODE_FOR_TEST && 'force-native-shadow-mode',
     COMPAT && 'compat',
+    ENABLE_NATIVE_CUSTOM_ELEMENT_LIFECYCLE && 'native-lifecycle',
+    ENABLE_SCOPED_CUSTOM_ELEMENT_REGISTRY && 'scoped-registry',
+    DISABLE_ARIA_REFLECTION_POLYFILL && 'disable-aria-polyfill',
+    NODE_ENV_FOR_TEST,
 ].filter(Boolean);
```

Comment:
Turns out I was not updating this properly as I added new test options. This fixes it – coverage is combined correctly now.